packages/admin-manifest/src/index.js

```diff
@@ -0,0 +1,157 @@
+function addManifest( manifest ) {
+	const link = document.createElement( 'link' );
+	link.rel = 'manifest';
+	link.href = 'data:application/manifest+json,' + JSON.stringify( manifest );
```

Comment:
This is clever. The PWA plugin uses the REST API to serve the manifest, but using a `data:` URI simplifies things. Was there a specific reason for doing it inline?

Reply:
See https://github.com/WordPress/gutenberg/pull/33102#discussion_r665136753